cmd/lncli/commands.go

```diff
@@ -2685,17 +2685,17 @@ var queryRoutesCommand = cli.Command{
 			Usage: "the max number of routes to be returned",
 			Value: 10,
 		},
-		cli.Int64Flag{
-			Name:  "max_cltv_delay",
-			Usage: "the total absolute time lock limit across the intermediate path",
-			Value: 1440,
-		},
 		cli.Int64Flag{
 			Name: "final_cltv_delta",
 			Usage: "number of blocks the last hop has to reveal " +
 				"the preimage",
 			Value: routing.DefaultFinalCLTVDelta,
 		},
+		cli.Int64Flag{
+			Name:  "max_cltv_delay",
```

Comment:
this commit should be squashed with the one where this was originally introduced, or be a `fixup` commit.